content/docs/concepts/traffic-management/network/index.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Network Resilience and Testing
+description: Learn about the dynamic failure recovery features of Istio that you can configure to build tolerance for failing nodes, and to prevent cascading failures to other nodes.
+weight: 3
+keywords: [traffic-management, timeout, retry, circuit-breaker, health-check, fault-injection, fault-tolerance-library]
+---
+
+Istio provides opt-in failure recovery features that you can configure
+dynamically at runtime through the [Istio traffic management rules](/docs/concepts/traffic-management/routing/virtual-services/#routing-rules).
+With these features, the service mesh can tolerate failing nodes and Istio can
+prevent localized failures from cascading to other nodes:
+
+-  **Timeouts and retries**
+
+    A timeout is the amount of time that Istio waits for a response to a
+    request. A retry is an attempt to complete an operation multiple times if
+    it fails. You can set defaults and specify request-level overrides for both
+    timeouts and retries or for one or the other.
+
+-  **Circuit breakers**
+
+    Circuit breakers prevent your application from stalling as it waits for an
+    upstream service to respond. You can configure a circuit breaker based on a
+    number of conditions, such as connection and request limits.
+
+-  **Health checks**
```

Comment:
remove health checks. we dont support it in the rules

Reply:
@rshriram This page is not about the routing rules, these is about the network resilience and testing features Istio, together with the Envoy proxies, enables.

Reply:
Shriram's point is that we have no way to configure health checks in Istio, so we shouldn't talk about it.

Reply:
@rshriram @frankbu Understdood. Thank you both. I'm removing the health checking information then.